docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>Do you think that this will be an issue for non-normalized Euclidean distance or other p-norm distances?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Yes, it would be great if you could add the derivation for this

Reply:
Sure. I will add it to the notebook in next set of commits.